test/e2e/e2e_test.go

```diff
@@ -38,7 +35,7 @@ type cliParams struct {
 var params cliParams
 
 func init() {
-	flag.StringVar(&params.ImageName, "image", "livepeer/catalyst", "Docker image to use when loading container")
+	flag.StringVar(&params.ImageName, "image", "livepeerci/catalyst:main", "Docker image to use when loading container")
```

Comment:
Hmmm, I think this will break the workflow of `make docker` then running the tests, because `make docker` currently builds `livepeer/catalyst`. Should either keep this as-is or update `Makefile` to make this new name

Reply:
Fixed